docs/guides/chip_tool_guide.md

```diff
@@ -868,3 +868,97 @@ In this command:
 -   _<node_id\>_ is the ID of the node that is going to receive the binding.
 -   _<endpoint_id\>_ is the ID of the endpoint on which the `binding` cluster is
     implemented.
+
+### Multi-admin scenario
+
+Multi-admin feature allows you to join Matter device to several Matter fabrics
+and administer it by several different Matter administrators. First you need to
+commission the Matter device to first fabric following the
+[Using CHIP Tool for Matter device testing](#using) section.
+
+Before it is possible to commission a Matter device to a new fabric, the
+administrator from first fabric must open the commissioning window for a new
+administrator from another fabric.
+
+To open the commissioning window on the paired Matter device, use the following
+command pattern:
+
+```
+$ ./chip-tool pairing open-commissioning-window <node_id> <option> <window_timeout> <iteration> <discriminator>
+```
+
+In this command:
+
+-   _<node_id\>_ is the ID of the node that should open commissioning window
+-   _<option\>_ is equal to 1 for Enhanced Commissioning Method and 0 for Basic
+    Commissioning Method
+-   _<window_timeout\>_ is time in seconds, before the commissioning window
+    closes
+-   _<iteration\>_ is number of PBKDF iterations to use to derive the PAKE
+    verifier
+-   _<discriminator\>_ is device specific value determined during commissioning
```

Comment:
iteration and discriminator are both ignored when option is set to 0.  We might want to document that.